Resources/Prototypes/Catalog/Fills/Lockers/security.yml

```diff
@@ -194,11 +185,12 @@
   - type: StorageFill
     contents:
       - id: ClothingEyesGlassesSecurity
-        prob: 0.3
-      - id: ClothingHeadHatFedoraBrown
-      - id: ClothingNeckTieDet
-      - id: ClothingOuterVestDetective
-      - id: ClothingOuterCoatDetective
+#        prob: 0.3 # DeltaV - standardise until HUD balance worked out
+#      - id: ClothingHeadHatFedoraBrown # DeltaV - removed for incongruence
+#      - id: ClothingNeckTieDet # DeltaV - removed for incongruence
+#      - id: ClothingOuterVestDetective # DeltaV - removed for incongruence
+#      - id: ClothingOuterCoatDetective # DeltaV - removed for incongruence
```

Comment:
Let people decide how they want their characters to look. Net negative of content.

Reply:
Feedback noted